yarn-project/prover-client/src/orchestrator/orchestrator.ts

```diff
@@ -66,6 +75,11 @@ enum PROMISE_RESULT {
   OPERATIONS,
 }
 
+const KernelTYpesWithoutFunctions: Set<PublicKernelType> = new Set<PublicKernelType>([
```

Comment:
small typo (this will require changes lower in the file too)
```suggestion
const KernelTypesWithoutFunctions: Set<PublicKernelType> = new Set<PublicKernelType>([
```